src/components/Todo.js

```diff
@@ -1,6 +1,10 @@
-import React from "react";
+import React, { useState } from "react";
+import DetailsForm from './DetailsForm'
```

Comment:
the component shall be imported before use, and follow the TitleCase while using and importing components. @raiab